bip-taproot.mediawiki

```diff
@@ -141,7 +141,7 @@ As the message for signature verification, transaction digest is ''hash<sub>TapS
 ** If the <code>SIGHASH_SINGLE</code> flag is set:
 *** <code>sha_single_output</code> (32): the SHA256 of the corresponding output in <code>CTxOut</code> format.
 
-The total number of bytes hashed is at most ''211''<ref>'''What is the number of bytes hashed for the signature hash?''' The total size of the input to ''hash<sub>TapSighash</sub>'' (excluding the initial 64-byte hash tag) can be computed using the following formula: ''178 - is_anyonecanpay * 52 - is_none * 32 + has_annex * 32''.</ref>.
+The total number of bytes hashed at top level is at most ''211''<ref>'''What is the number of bytes hashed for the signature hash?''' The total size of the input to ''hash<sub>TapSighash</sub>'' (excluding the initial 64-byte hash tag) can be computed using the following formula: ''178 - is_anyonecanpay * 52 - is_none * 32 + has_annex * 32''.</ref>.  Sub-hashes, such as `sha_prevouts`, may be cached across signatures of the same transaction.
```

Comment:
I think adding `at top level` is at least as confusing as before because it's not immediately clear what it means. Perhaps better is "The total number of bytes hashed is at most ''211'' (excluding sub-hashes like `sha_prevouts`)"?

Reply:
I like that.  Fixing